lib/bugsnag/session_tracker.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Bugsnag.SessionTracker do
+  import Supervisor.Spec
```

Comment:
Elixir docs say Supervisor.Spec is deprecated. I need to read about Supervisor and Task.Supervisor stuff so I can figure out this reporting.